.github/workflows/merged.yaml

```diff
@@ -0,0 +1,25 @@
+name: Merge Event
+
+on:
+  pull_request:
+    types:
+      - closed
+
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
+    runs-on: ubuntu-latest
+    steps:
+    - name: Create Mattermost Message
+      #https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#using-an-intermediate-environment-variable
+      env: 
+        NUMBER: ${{ github.event.number }}
+        PR_URL: https://github.com/${{ github.repository }}/pull/${{ github.event.number }}
+        REPO: ${{ github.repository }}
+        USER: ${{ github.actor }}
+        TITLE: ${{ github.event.pull_request.title }}
+      run: |
+        echo "{\"text\":\"[$REPO] | [$TITLE #$NUMBER]($PR_URL) was merged into master by $USER.\"}" > mattermost.json
```

Comment:
Used as an `ENV` variable, yes. It seems I also put the wrong link. I meant to put [this](https://docs.github.com/en/actions/security-guides/security-hardening-for-github-actions#example-of-a-script-injection-attack). Which describes this exact scenario.

Reply:
I can test the JSON structure though! Good catch.

Reply:
Turns out even though the quotes are fine security wise, it _does_ blow up the JSON parsing. Since this is a shell afterall I will add in a shorthand `sed` to cover that